src/wombats/daos/game.clj

```diff
@@ -105,7 +112,7 @@
     (let [frame-tmp-id (d/tempid :db.part/user)
           frame-trx {:db/id frame-tmp-id
                      :frame/frame-number 0
-                     :frame/round-number 0
+                     :frame/round-number 1
```

Comment:
Yeah, you can never start a game in round 0. Not a huge fan but it causes bugs when it tries to process the first round.